### PR TITLE
MuSiC bulk and compare tutorial

### PR DESCRIPTION
**Pull request by nomadscientist**
These tutorials were built off the PR about splitting transcriptomics into single cell and transcriptomics pages, so while these are in draft form that PR probably needs finishing first!

Also, having trouble with Gitpod hating me so haven't been able to render these, so let's call them a draft at best :(

### CODE REVIEW

**Comment by shiltemann** [comment]
might be nice to add a news post for these new tutorials? and possibly one announcing the new dedicated single-cell topic? instructions: https://training.galaxyproject.org/training-material/faqs/gtn/gtn_news_create_post.html

**Review by mtekman — DISMISSED** [review]
The changes look good @nomadscientist ! 
I agree that it makes sense to split out the last multi-sample deconvolution workflow for a different PR. 

I'm not sure what to do about the two missing links in the HTML proofer.

File: topics/single-cell/tutorials/bulk-music-3-preparebulk/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
+---
+layout: tutorial_hands_on
+subtopic: deconvo
+priority: 3
+
+title: Creating the bulk RNA-seq dataset for deconvolution
+zenodo_link: ''
```
Comment (shiltemann) [inline topics/single-cell/tutorials/bulk-music-3-preparebulk/tutorial.md:7]: The zenodo link is missing here, so links in the import data section are invalid.

**Comment by shiltemann** [comment]
looks good @nomadscientist! Just add the missing zenodo link to fix the failing tests and I think this is good to go :+1:

File: topics/single-cell/tutorials/bulk-music-3-preparebulk/tutorial.md (hunk 1 above)
Comment (nomadscientist) [reply topics/single-cell/tutorials/bulk-music-3-preparebulk/tutorial.md]: thanks, doh, should've caught that, sorry!

File: topics/single-cell/tutorials/bulk-music-3-preparebulk/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
+---
+layout: tutorial_hands_on
+subtopic: deconvo
+priority: 3
+
+title: Creating the bulk RNA-seq dataset for deconvolution
+zenodo_link: '7319173'
```
Comment (shiltemann) [inline topics/single-cell/tutorials/bulk-music-3-preparebulk/tutorial.md:7]:
```suggestion
zenodo_link: 'https://zenodo.org/record/7319173'
```

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
Thanks @nomadscientist and @mtekman !! :tada:

**Pull request merged**